views/default/manage.pdt

```diff
@@ -1,69 +1,65 @@
+    <h3><?php $this->_('CWatch.manage.title'); ?></h3>
 </div>
 
-    <?php
-	$num_rows = count($this->Html->ifSet($module->rows));
-	
-	if ($num_rows > 0) {
-	?>
+<?php
+$num_rows = count($this->Html->ifSet($module->rows));
 
+if ($num_rows > 0) {
+?>
 <table class='table'>
     <tr class='heading_row'>
-        <td><span><?php $this->_('CWatch.heading_username.name'); ?></span></td>
-        <td><span><?php $this->_('CWatch.heading_options.name'); ?></span></td>
+        <td><span><?php $this->_('CWatch.manage.heading_username'); ?></span></td>
+        <td><span><?php $this->_('CWatch.manage.heading_options'); ?></span></td>
     </tr>
 
-	<?php
-	    for ($i=0; $i < $num_rows; $i++)
-	    {
-	?>
+    <?php
+    for ($i = 0; $i < $num_rows; $i++) {
+    ?>
     <tr<?php echo ($i % 2 == 1) ? ' class="odd_row"' : ''; ?>>
         <td><?php $this->Html->_($module->rows[$i]->meta->username); ?> </td>
         <td>
-            <a href="<?php echo $this->Html->safe($this->base_uri . 'settings/company/modules/editrow/' . $this->Html->ifSet($module->id) . '/' . $this->Html->ifSet($module->rows[$i]->id) . '/'); ?>"><?php $this->_('CWatch.option_edit'); ?></a>  
-                    <?php
-                        $this->Form->create($this->base_uri . 'settings/company/modules/deleterow/');
-                        $this->Form->fieldHidden('id', $this->Html->ifSet($module->id));
-                        $this->Form->fieldHidden('row_id', $this->Html->ifSet($module->rows[$i]->id));
-                    ?>
-
-            <a href="<?php echo $this->Html->safe($this->base_uri . 'settings/company/modules/deleterow/' . $this->Html->ifSet($module->id) . '/' . $this->Html->ifSet($module->rows[$i]->id) . '/'); ?>" class="manage" rel="<?php echo $this->Html->safe($this->_('CWatch.manage.module_row.confirm_delete', true)) . $module->rows[$i]->meta->username; ?>"><?php $this->_('CWatch.manage.module_row.delete'); ?></a>                    
-		    <?php
-                    $this->Form->end();
-                    ?>
+            <a href="<?php echo $this->Html->safe($this->base_uri . 'settings/company/modules/editrow/' . $this->Html->ifSet($module->id) . '/' . $this->Html->ifSet($module->rows[$i]->id) . '/'); ?>"><?php $this->_('CWatch.manage.option_edit'); ?></a>
 
+            <?php
+            $this->Form->create($this->base_uri . 'settings/company/modules/deleterow/');
+            $this->Form->fieldHidden('id', $this->Html->ifSet($module->id));
+            $this->Form->fieldHidden('row_id', $this->Html->ifSet($module->rows[$i]->id));
+            ?>
+            <a href="<?php echo $this->Html->safe($this->base_uri . 'settings/company/modules/deleterow/' . $this->Html->ifSet($module->id) . '/' . $this->Html->ifSet($module->rows[$i]->id) . '/'); ?>" class="manage" rel="<?php echo $this->Html->safe($this->_('CWatch.manage.confirm_delete', true)) . $module->rows[$i]->meta->username; ?>"><?php $this->_('CWatch.manage.delete'); ?></a>
```

Comment:
The rel= attribute makes the language definition HTML-safe but not the username. We usually assume that language definitions are HTML-safe, but I would expect at least the username to be output HTML-safe